.github/workflows/Build.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Set workflow & branch variables
         id: set-variables
         run: |
-          export TIMESTAMP="$(date -u +%F-%H-%M)"
+          export TIMESTAMP="$(date -u +%F-%H)Z"
```

Comment:
I guess I'm not understanding the `Z`.

Reply:
Agreed. Meant to be zulu, but adjusting to just show TZ...